src/gui/qgsmediawidget.cpp

```diff
@@ -126,8 +126,8 @@ int QgsMediaWidget::videoHeight() const
 
 void QgsMediaWidget::setVideoHeight( int height )
 {
-  mVideoWidget->setMinimumHeight( height );
-  mVideoWidget->setMaximumHeight( height > 0 ? height : 9999 );
+  mVideoWidget->setMinimumHeight( height > 0 ? height : 250 );
+  mVideoWidget->setMaximumHeight( height > 0 ? height : 250 );
```

Comment:
could this be an issue if there is a spliiter?

Reply:
Na, has to do with QVideoWidget wrongly setting its sizeHint to QSize( 0, 0 ) when restarting video after first run through.